### ISSUE TITLE
OSCAR Package Configuration GUI bug

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#168]
**Reported by bli on 14 Mar 2006 20:18 UTC**
```
When the user clicks on a package to configure, the
parent "OSCAR Package Configuration" dialog is not
frozen such that it is possible to click on another
package thus destroying the original package
configuration dialog.  This IMHO is a bug and needs to
be fixed.
```

**Comment by dikim33** [comment]
**Comment by muglerj on 21 Mar 2006 18:28 UTC**
```

Moved to 5
```

**Comment by dikim33** [comment]
**Modified by jparpail on 6 Apr 2007 13:16 UTC**

**Comment by dikim33** [comment]
**Comment by jparpail on 10 Apr 2007 09:17 UTC**
Oops,
Reverting to version "branch-4-2"

**Comment by dikim33** [comment]
**Modified by jparpail on 20 Jul 2007 13:53 UTC**

**Comment by dikim33** [comment]
**Comment by dikim on 6 Jan 2008 22:54 UTC**
(In [6818]) I hope this check-in fixes #170 and this works fine for everyone else
too.
